app/src/main/javascript/bundles/experiment-page/src/ExperimentPageRouter.js

```diff
@@ -10,130 +10,169 @@ import SupplementaryInformationRoute from './SupplementaryInformationRoute'
 import DownloadsRoute from './DownloadsRoute'
 
 const RoutePropTypes = {
-  match: PropTypes.object.isRequired,
-  location: PropTypes.object.isRequired,
-  history: PropTypes.object.isRequired
+    match: PropTypes.object.isRequired,
```

Comment:
To clean the code and make this file smaller, I would externalise all the `proptypes` to another file.